sdk/compute/Azure.ResourceManager.Compute/api/Azure.ResourceManager.Compute.net8.0.cs

```diff
@@ -5575,6 +5608,18 @@ protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer
         string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Compute.Models.EncryptionSettingsGroup>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
         System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Compute.Models.EncryptionSettingsGroup>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
     }
+    public partial class EventGridAndResourceGraph : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Compute.Models.EventGridAndResourceGraph>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Compute.Models.EventGridAndResourceGraph>
+    {
+        public EventGridAndResourceGraph() { }
+        public bool? Enable { get { throw null; } set { } }
```

Comment:
I think we should rename this to `IsEnabled`.

Reply:
Added this transformation but not sure why it didn't work:
`  EventGridAndResourceGraph.Enable: IsEventGridAndResourceGraphEnabled`